### PR TITLE
BTP_SM_ADMIN_CREDS env to reuse SM instance in skr test

### PR DESCRIPTION
**Pull request by tillknuesting**
BTP_SM_ADMIN_CREDS env to reuse SM instance in skr test

### CODE REVIEW

File: templates/templates/prow-config.yaml [hunk 1]
```diff
@@ -1033,6 +1033,34 @@ presets:
           secretKeyRef:
             name: btp-operator-credentials
             key: clusterid
+  - labels:
+      preset-btp-admin-stage-credentials: "true"
```
Comment (Halamix2) [inline templates/templates/prow-config.yaml:1037]:
```suggestion
        preset-btp-admin-stage-credentials: "true"
```

**Comment by Halamix2** [comment]
Instead of running ` go run development/tools/cmd/rendertemplates/main.go --config templates/config.yaml` you can also run `make jobs-definitions`

File: templates/templates/prow-config.yaml [hunk 2]
```diff
@@ -1033,6 +1033,34 @@ presets:
...
+        - name: BTP_SM_ADMIN_TOKENURL
+          valueFrom:
+            secretKeyRef:
+              name: btp-admin-credentials
+              key: tokenurl
+        - name: BTP_SM_ADMIN_CLUSTERID
+          valueFrom:
+            secretKeyRef:
+              name: btp-admin-credentials
+              key: clusterid
```
Comment (Halamix2) [inline templates/templates/prow-config.yaml:1063]:
```suggestion
  - labels:
      preset-btp-admin-stage-credentials: "true"
    env:
      - name: BTP_SM_ADMIN_CLIENTID
        valueFrom:
          secretKeyRef:
            name: btp-admin-credentials
            key: clientid
      - name: BTP_SM_ADMIN_CLIENTSECRET
        valueFrom:
          secretKeyRef:
            name: btp-admin-credentials
            key: clientsecret
      - name: BTP_SM_ADMIN_URL
        valueFrom:
          secretKeyRef:
            name: btp-admin-credentials
            key: url
      - name: BTP_SM_ADMIN_TOKENURL
        valueFrom:
          secretKeyRef:
            name: btp-admin-credentials
            key: tokenurl
      - name: BTP_SM_ADMIN_CLUSTERID
        valueFrom:
          secretKeyRef:
            name: btp-admin-credentials
            key: clusterid
```

**Review by PK85 — APPROVED** [review]

**Review by Halamix2 — APPROVED** [review]

**Pull request merged**